File: learn/data-structures/arrays/searching/only-repetitive-element-between-1-to-n-1/only-repetitive-element.js
function getRepeatedElement(arr, n) {
  let sum = 0;
  for (let i = 0; i < n; i++) {
    sum = sum + arr[i];
  }
  return sum - ((n - 1) * n) / 2;
}
let arr = [1, 5, 3, 2, 6, 4, 7, 8, 4];
let len = arr.length;
console.log("Repeated element: ", getRepeatedElement(arr, len));

File: learn/data-structures/arrays/searching/only-repetitive-element-between-1-to-n-1/only-repetitive-element.cpp
#include<iostream>

using namespace std;

int findRepetitive(int array[], int n){
    int sum = 0;
    for (int i = 0; i < n; i++){
        sum += array[i];
    }

    return (sum - (n*(n - 1)) / 2);
}

int main(){
    int arr[] = {1, 3, 2, 4, 5, 7, 6, 8, 4};
    int arr_size = sizeof(arr) / sizeof(arr[0]);
    cout << "Only repetitive element in array: " << findRepetitive(arr, arr_size) << endl;
    return 0;
}



